libs/judicial-system/types/src/lib/types.ts

```diff
@@ -14,6 +14,7 @@ export interface User {
   mobileNumber: string
   email: string
   role: UserRole
+  institution?: string
```

Comment:
Would it be a good idea to use an enum for this?

Reply:
I would like to handle this and the court in a similar way. Currently, courts are stored in cases as strings, so I would like this to be a string as well. We may consider changing this later, especially if used in drop downs and such in the client.